en/config/safety.md

```diff
@@ -0,0 +1,235 @@
+# Safety Configuration (Failsafes) 
+
+PX4 has configurable failsafe systems to protect and recover your vehicle if something goes wrong! These allow you to specify areas and conditions under which you can safely fly, and the [action](#failsafe_actions) that will be performed if a failsafe is triggered (for example, landing, holding position, or returning to a specified point).
+
+The most important failsafe settings are configured in the *QGroundControl* [Safety Setup](#qgc_safety_setup) page. Others must be configured via [parameters](#failsafe_other).
+
+## Failsafe Actions {#failsafe_actions}
+
+Each failsafe defines its own set of actions. Some of the more common failsafe actions are:
+
+Action | Description
+--- | --- 
+None/Disabled | No action (the failsafe will be ignored).
+Warning | A warning message will be sent to *QGroundControl*.
+Loiter | The vehicle will enter [Hold mode](../flight_modes/hold.md). For multicopters this means the vehicle will hover, while for fixed/wing the vehicle will circle.
+Return to Land | The vehicle will enter [Return mode](../flight_modes/return.md). Return behaviour can be set in the [Return Home Settings](#return_settings) (below).
+Land at current position |  The vehicle enters [Land mode](../flight_modes/land.md), and lands immediately.
+RC Auto Recovery (CASA Outback Challenge rules) | ?
+Terminate/Flight termination | ?
+Lockdown | ?
+
+
+> **Note** You can only specify the action for the *first* failsafe event. Once a failsafe occurs the system will enter special handling code, such that subsequent failsafe triggers are managed by separate system level and vehicle specific code. This might result in the vehicle being changed to a manual mode so the user can directly manage recovery.
```

Comment:
That is new to me. From where did you get this?

Reply:
It comes from discussion with PaulR [here](https://github.com/PX4/Firmware/issues/9336). I haven't got any tests to show it is true or not true. What I do know is that we do handle the first failsafe event, and if this is true then I don't have to try and document what happens in the event of failsafes occuring in failsafes.

Reply:
Ok. I think it depends on the failure case: from what I know if you lose RC, and the vehicle goes into RTL, and then you regain RC, you can simply switch modes and regain control. Subsequent RC losses will be handled the same way. But this should quickly be verified.

Reply:
Here we aren't actually talking about recovery cases - this is more what would happen say if you had RC loss followed immediately by battery loss.

The confusion I have is more from cases like RC loss followed by position loss. This looks like 2 failsafes (so should be higher level system handling), but the actual handling seems more like it is defined by what happens in COM_POSCTL_NAVL

What you're saying above is almost certainly true - ie that if you can recover from failsafe (ie you get RC back)  then everything can be reset by the user changing mode. Worth saying?